examples/exotica_examples/scripts/example_point_to_line.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+import rospy
+import numpy as np
+import pyexotica as exo
+from pyexotica.publish_trajectory import *
+import task_map_py
+from sensor_msgs.msg import Joy, JointState
+from visualization_msgs.msg import Marker
+from std_msgs.msg import ColorRGBA as Color
+
+green=Color()
+green.r=0.0
+green.g=1.0
+green.b=0.0
+green.a=1.0
+
+DT = 1.0/100.0 # 100 HZ
+DAMP = 0.005
+
+INIT_POSITION = np.array([0.75, 0, 0.5]) # initial position of the end point
+
+def MarkerMsg(p):
+    marker=Marker()
+    marker.type=marker.SPHERE
+    marker.id=0
+    marker.action=marker.ADD
+    marker.header.frame_id='exotica/world_frame'
+    for i, d in enumerate(['x', 'y', 'z']):
+        setattr(marker.pose.position, d, p[i])
+        setattr(marker.pose.orientation, d, 0.0)
+        setattr(marker.scale, d, 0.15)
+    marker.pose.orientation.w=1.0
+    marker.color=green
+    return marker
+
+class Example(object):
+
+    def __init__(self):
+
+        exo.Setup.initRos()
+        self.solver = exo.Setup.loadSolver('{exotica_examples}/resources/configs/point_to_line.xml')
+        self.problem = self.solver.getProblem()
+        self.q=np.array([0.0]*7)        
+
+        self.p = INIT_POSITION
+
+        self.pub = {}
+        self.pub['marker']=rospy.Publisher('/POINT', Marker, queue_size=1)
```

Comment:
Don't use the root-named topic names (as it prevents putting nodes into groups). Always use local topic names, i.e., ``POINT`` here [but I'd recommend using only lower case] and ``joint_states``.